system-parachains/asset-hubs/asset-hub-kusama/src/constants.rs

```diff
@@ -1,106 +0,0 @@
-// Copyright (C) 2021 Parity Technologies (UK) Ltd.
-// SPDX-License-Identifier: Apache-2.0
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// 	http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-pub mod currency {
-	use kusama_runtime_constants as constants;
-	use polkadot_core_primitives::Balance;
-
-	/// The existential deposit. Set to 1/100 of its parent Relay Chain.
-	pub const EXISTENTIAL_DEPOSIT: Balance = constants::currency::EXISTENTIAL_DEPOSIT / 100;
-
-	pub const UNITS: Balance = constants::currency::UNITS;
-	pub const CENTS: Balance = constants::currency::CENTS;
-	pub const GRAND: Balance = constants::currency::GRAND;
-	pub const MILLICENTS: Balance = constants::currency::MILLICENTS;
-
-	pub const fn deposit(items: u32, bytes: u32) -> Balance {
-		// map to 1/100 of what the kusama relay chain charges (v9020)
-		constants::currency::deposit(items, bytes) / 100
-	}
-}
-
-/// Fee-related.
-pub mod fee {
-	use frame_support::weights::{
-		constants::ExtrinsicBaseWeight, FeePolynomial, Weight, WeightToFeeCoefficient,
-		WeightToFeeCoefficients, WeightToFeePolynomial,
-	};
-	use polkadot_core_primitives::Balance;
-	use smallvec::smallvec;
```

Comment:
I think that now we could remove `smallvec = "1.11.0"` from all touched runtimes, e.g: https://github.com/joepetrowski/runtimes/blob/actually-fix-ed/system-parachains/asset-hubs/asset-hub-kusama/Cargo.toml#L16C1-L16C20